internal/lib/lib.go

```diff
@@ -176,34 +185,40 @@ func GetModelName(namespace, objectName string) string {
 
 // All L4 object names.
 func GetL4VSName(svcName, namespace string) string {
-	return Encode(NamePrefix + namespace + "-" + svcName)
+	return Encode(NamePrefix+namespace+"-"+svcName, "L4 Virtual Service ")
```

Comment:
why the trailing space in the end of the objType strings? also should we keep it single words or better constants?